radix-engine/src/vm/wasm_runtime/no_op_runtime.rs

```diff
@@ -182,8 +185,11 @@ impl<'a> WasmRuntime for NoOpWasmRuntime<'a> {
         Err(InvokeError::SelfError(WasmRuntimeError::NotImplemented))
     }
 
-    fn consume_gas(&mut self, n: u32) -> Result<(), InvokeError<WasmRuntimeError>> {
-        self.gas_consumed.add_assign(n);
+    fn consume_wasm_execution_units(
+        &mut self,
+        n: u32,
+    ) -> Result<(), InvokeError<WasmRuntimeError>> {
+        self.wasm_execution_units_consumed.add_assign(n);
```

Comment:
We just saw a panic here in tests - perhaps we should catch / handle this as a fee reserve overflow?